lib/search/format-package-stream.js

```diff
@@ -150,6 +150,8 @@ function highlightSearchTerms (str, terms) {
 
 function normalizePackage (data, opts) {
   opts = opts || {}
+  var version = Object.keys(data.versions || {})[0] || []
+  if (data.version) version = data.version
```

Comment:
These two lines are basically unnecessary -- both `esearch` and `all-package-search` return a consistent data format (specified at the top of this file). The patch itself just needs to be `version: data.version`, unless I brain farted hard.